itou/www/apply/views/list_views.py

```diff
@@ -469,9 +469,11 @@ def list_for_siae_actions(request):
         "can_process": can_process,
         "can_postpone": can_postpone,
         "can_refuse": can_refuse,
-        "can_transfer": can_transfer,
-        "enable_transfer": enable_transfer,
-        "other_actions_count": sum([can_add_to_pool, can_process, can_postpone, can_archive, can_transfer]),
+        "can_transfer_internal": can_transfer_internal,
+        "can_transfer_external": can_transfer_external,
+        "other_actions_count": sum(
+            [can_add_to_pool, can_process, can_postpone, can_archive, can_transfer_internal, can_transfer_external]
```

Comment:
```suggestion
            [can_add_to_pool, can_process, can_postpone, can_archive, can_transfer_internal or can_transfer_external]
```
Les deux transferts apparaissent comme la même action non ? Et cela pourrait peut-être mériter un commentaire.